node/public/admin-dash/index.html

```diff
@@ -57,7 +57,7 @@ <h2 class="stsadmin-subheader">Update Product</h2>
                     <input onclick="" class="submit-btn grn" type="submit" value="Submit"></input>
                   </form>
                   <p id="successAlert">Alert</p>
-                  <a class="back-to" href="https://stshosting-stesse.wedeploy.io/pages/stsadmin.html#manage-products"><p><-- back to main</p></a>
```

Comment:
same here.

Reply:
Changing to a relative path now because of migrate